### ISSUE TITLE
Thank you for this sample

### ISSUE THREAD

**Issue by florianwachs** [RadekVyM/MarvelousMAUI#1]
I canceled twitter so I have to write here. Thank you very much for this sample and your OSS contributions.
Have a great day.

**Comment by RadekVyM** [comment]
Thanks @florianwachs, I really appreciate your message. It motivates me to continue sharing my work 🙂

**Comment by manishbisen** [comment]
Hey @RadekVyM,  This is awesome. Please add this sample in maui sample list so that it can be reach out to more .net developers.

**Comment by AswinPG** [comment]
@RadekVyM You're just awesome. This sample is unbelievable. Thanks a lot for your contributions, especially SimpleShell.

**Comment by codeanees** [comment]
This is really the Best MAUI project I ever seen.  I compared both Flutter and MAUI version. I don't see any glitch and animations are butter like smooth. 

The best part is you never give-up on any of the items from Flutter version.  I see each and every single piece been cloned in MAUI version.  I really like to know more about the huddles you faced during developing and how you overcome it.

**Comment by RadekVyM** [comment]
@manishbisen, @AswinPG, @codeanees

Thanks for the support.

**Comment by RadekVyM** [comment]
@codeanees

Actually, there are a few details that I missed (I want to look into some of them in the future). For example, I was not able to find workarounds for some bugs with `CarouselView` and had to make compromises.

**Comment by FM1973** [comment]
@RadekVyM 
Oh man, thanks a lot! I wish Microsoft would provide high quality samples like this. Keep up the good work!

**Comment by osardar** [comment]
Agreed- this is extremely helpful, and probably the best .NET MAUI example I've seen so far. Thank you so much for sharing this!
